changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 24th October 2023
+
+* Added [Exports](../operations/exports.md) operations.
```

Comment:
```suggestion
* Added [Exports](../operations/exports.md) operations:
  * [Get all exports](../operations/exports.md#get-all-exports)
  * [Add export](../operations/exports.md#add-export)
```
This should be the way we we normally write it.
Only question would be, if we should mention here that it is in beta testing and when we remove restricted, we should make another changelog entry. I would probably mention it, because then partners are aware that it is no longer restricted and they can start using it freely.

Reply:
Agreed, will add mention of beta test to the entry.